site/specs-source/voice.json

```diff
@@ -1333,7 +1333,7 @@
           "Recordings"
         ],
         "summary": "Delete Transcription",
-        "description": "Deletes the specified recording's transcription.",
+        "description": "Deletes the specified recording's transcription.\n\nNote: After the deletion is requested and a `204` is returned, the transcription will not be accessible anymore. However, it is not deleted immediately. This deletion process, while transparent and irreversible, can take an additional 24 to 48 hours.",
```

Comment:
This came up in another PR-- I don't think it's super necessary to talk about the deletion process if after the 204 the recording is inaccessible to the customer anyway. I think telling them it may take longer may lead to customers thinking there's some 'safety net' and if they accidentally delete something they can just talk to us to get it back. But that's just my opinion

Reply:
Everyone except Nick thinks that I think. Nick added this